src/program/services/downloaders/models.py

```
@@ -0,0 +1,113 @@
+
+
+class DebridFile(BaseModel):
+    """Represents a file in from a debrid service"""
+    file_id: Optional[int] = Field(default=None)
+    filename: Optional[str] = Field(default=None)
+    filesize: Optional[int] = Field(default=None)
+
+    @classmethod
+    def create(cls, filename: str, filesize_bytes: int, filetype: Literal["movie", "episode"], file_id: Optional[int] = None) -> Optional["DebridFile"]:
+        """Factory method to validate and create a DebridFile"""
+        if not any(filename.endswith(ext) for ext in VIDEO_EXTENSIONS) and not "sample" in filename.lower():
```

Ruff (0.8.0)

Test for membership should be `not in`

Convert to `not in`

(E713)